openupgrade_scripts/scripts/analytic/16.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,94 @@
+---Models in module 'analytic'---
+obsolete model account.analytic.distribution
+obsolete model account.analytic.tag
+new model account.analytic.applicability
+new model account.analytic.distribution.model
+new model analytic.mixin [abstract]
+# NOTHING TO DO
```

Comment:
This analysis work file is base on newly generation at PR : https://github.com/OCA/OpenUpgrade/pull/4042. I did not edit the original `upgrade_analysis` only `ugrade_analysis_work`